server/portal/apps/projects/management/commands/projects_id.py

```diff
@@ -1,96 +1,72 @@
 """Management command."""
 
 from django.conf import settings
+from django.db import transaction
+from django.core.exceptions import ObjectDoesNotExist
 from django.core.management.base import BaseCommand
 from portal.libs.agave.utils import service_account
-from portal.libs.agave.models.files import BaseFile
 from portal.apps.projects.models.base import ProjectId, Project
+from portal.libs.agave.operations import iterate_listing
 
 
 class Command(BaseCommand):
-    """Command class."""
-
     help = (
-        'Manage projects next Id. By default this command will print '
+        'Manage projects latest project i. By default this command will print '
         'the current latest project id, the last project id used in '
-        'storage systems and the last project id used in folders created '
-        'in corral.'
+        'storage systems and the last project id used in folders created.'
     )
 
-    def _get_latest_project_storage(self):  # pylint:disable=no-self-use
-        """Get latest agave project storage."""
+    @staticmethod
+    def get_latest_project_storage(ignore_above_value=None):
+        """Get latest agave project storage.
+
+        :param ignore_above_value: If provided, then ignore projects ids that are greater than or equal to this value.
```

Comment:
would `max_value` or `max_project_id` be a better param name here, instead of `ignore_above_value`?